CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,182 @@
+# Code of Conduct at nf-core (v1.4)
```

Comment:
this file is only in template for nf-core pipelines

Reply:
Done